openhcl/virt_mshv_vtl/src/processor/hardware_cvm/mod.rs

```diff
@@ -343,9 +343,9 @@ impl<T, B: HardwareIsolatedBacking> hv1_hypercall::SetVpRegisters
 {
     fn set_vp_registers(
         &mut self,
+        vtl: Option<Vtl>,
```

Comment:
Ah, sorry to be a pain. In this particular case, we want to keep vtl where it was. You can think about these parameters as going from least to most specific: we're talking about X partition, Y VP (in the context of partition X), Z VTL (in the context of VP Y).